pkg/config/dns-server/config.go

```diff
@@ -9,12 +9,14 @@ import (
 
 // DNS Server configuration
 type DNSServerConfig struct {
-	// The domain that the server will resolve the services for
+	// Domain The domain that the server will resolve the services for
 	Domain string `yaml:"domain" envconfig:"kuma_dns_server_domain"`
 	// Port on which the server is exposed
 	Port uint32 `yaml:"port" envconfig:"kuma_dns_server_port"`
 	// CIDR used to allocate virtual IPs from
 	CIDR string `yaml:"CIDR" envconfig:"kuma_dns_server_cidr"`
+	// v2CIDR used to allocate virtual IPs from
```

Comment:
Actually is `CIDRv2` the right name for this? I kind of ran out of idea on naming this :)

Reply:
Myabe it's virtualOutbound cidr ?

Reply:
I just realized `v2` is not because of Envoy's `v2` and `v3` :) I agree then, the name has to be changed. I have 2 questions regarding this:
1. Why do we need the second CIDR? Only for the user's convenience or from the technical point of view it's not possible to share the same CIDR?
2. If we need 2 CIDRs can we split them internally? I mean we can let user configure `240.0.0.0/8` and we split it into `240.0.0.0/9` and `240.128.0.0/9` (I hope I didn't mess up with this example)

Reply:
(who doesn't mess up with cidr :))
Yes we could do internally the split I do.
I guess I was a little lazy and didn't want to handle the range split myself.
The reason we need 2 CIDR is because in v1 this cidr is used globally for a mesh. In v2 it's only for a Dataplane. So it was tricky to have the new version need the centralisation required by v1.